src/panels/lovelace/elements/hui-state-label-element.ts

```diff
@@ -0,0 +1,78 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/entity/ha-state-label-badge.js";
+
+import { computeTooltip } from "../../../common/string/compute-tooltip";
+import computeStateDisplay from "../../../common/entity/compute_state_display.js";
+import { handleClick } from "../../../common/dom/handle-click";
+import { longPress } from "../common/directives/long-press-directive";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceElement, LovelaceElementConfig } from "./types.js";
+import { HomeAssistant } from "../../../types.js";
+import { TemplateResult } from "lit-html";
+
+interface Config extends LovelaceElementConfig {
+  prefix?: string;
+  suffix?: string;
+}
+
+class HuiStateLabelElement extends hassLocalizeLitMixin(LitElement)
+  implements LovelaceElement {
+  public hass?: HomeAssistant;
+  private _config?: Config;
+
+  static get properties() {
+    return { hass: {}, _config: {} };
+  }
+
+  public setConfig(config: Config): void {
+    if (!config.entity) {
+      throw Error("Invalid Configuration: 'entity' required");
+    }
+
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config) {
+      return html``;
+    }
+
+    const state = this.hass!.states[this._config.entity!];
+    return html`
+      ${this.renderStyle()}
+      <div
+        .title="${computeTooltip(this.hass!, this._config)}"
+        @ha-click="${() => handleClick(this, this.hass!, this._config!, false)}"
+        @ha-hold="${() => handleClick(this, this.hass!, this._config!, true)}"
+        .longPress="${longPress()}"
+      >
+      ${this._config.prefix}${
+      state ? computeStateDisplay(this.localize, state) : "-"
+    }${this._config.suffix}
```

Comment:
This doesnt look very Pretty? Do you have prettier format on save?

Reply:
I have the VS Code extension "Prettier" which gave me this 🤷‍♂️

Reply:
Prettier doesn't understand that this template literal contains HTML, so it will reformat it so that it is still representing the same AST, which makes it look pretty weird. They are working on HTML Support 👍